java/code/src/com/suse/manager/model/hub/IssHub.java

```diff
@@ -0,0 +1,151 @@
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "suseISSHub")
+public class IssHub extends BaseDomainHelper {
+    private Long id;
+    private String fqdn;
+    private String rootCa;
+    private SCCCredentials mirrorCredentials;
+
+    protected IssHub() {
+        // Default empty Constructor for Hibernate
+    }
+
+    /**
+     * Constructor
+     * @param fqdnIn the FQDN of the Hub Server
+     */
+    public IssHub(String fqdnIn) {
+        this (fqdnIn, null);
+    }
+
+    /**
+     * Constructor
+     * @param fqdnIn the FQDN of the Hub Server
+     * @param rootCaIn the root CA used by the Hub Server
+     */
+    public IssHub(String fqdnIn, String rootCaIn) {
+        fqdn = fqdnIn;
+        rootCa = rootCaIn;
+        mirrorCredentials = null;
+    }
+
+    /**
+     * @return return the ID
+     */
+    @Id
+    @Column(name = "id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    public Long getId() {
+        return id;
+    }
+
+    /**
+     * Get the FQDN of the Hub Server
+     * @return return the FQDN of the Hub Server
+     */
+    @Column(name = "fqdn", unique = true)
+    public String getFqdn() {
+        return fqdn;
+    }
+
+    /**
+     * Get the configured Root CA
+     * @return return the root ca
+     */
+    @Column(name = "root_ca")
+    public String getRootCa() {
+        return rootCa;
+    }
+
+    /**
+     * Get the mirror credentials.
+     * @return the credentials
+     */
+    @ManyToOne(targetEntity = SCCCredentials.class)
+    @JoinColumn(name = "mirror_creds_id")
+    public SCCCredentials getMirrorCredentials() {
+        return mirrorCredentials;
+    }
+
+    /*
+     * will be autogenerated
+     */
+    protected void setId(Long idIn) {
+        id = idIn;
+    }
+
+    /**
+     * @param fqdnIn the FQDN
+     */
+    public void setFqdn(String fqdnIn) {
+        fqdn = fqdnIn;
+    }
+
+    /**
+     * @param rootCaIn the root ca
+     */
+    public void setRootCa(String rootCaIn) {
+        rootCa = rootCaIn;
+    }
+
+    /**
+     * @param mirrorCredentialsIn the mirror credentials
+     */
+    public void setMirrorCredentials(SCCCredentials mirrorCredentialsIn) {
+        mirrorCredentials = mirrorCredentialsIn;
+    }
+
+    @Override
+    public boolean equals(Object oIn) {
+        if (this == oIn) {
+            return true;
+        }
+        if (!(oIn instanceof IssHub issHub)) {
+            return false;
+        }
+        return Objects.equals(getFqdn(), issHub.getFqdn()) &&
+                Objects.equals(getRootCa(), issHub.getRootCa()) &&
+                Objects.equals(getMirrorCredentials(), issHub.getMirrorCredentials());
+    }
+
+    @Override
+    public int hashCode() {
+        return new HashCodeBuilder().append(getFqdn()).append(getRootCa()).append(getMirrorCredentials()).toHashCode();
```

Comment:
JFYI, since you haven't used Apache Commons for `equals()` and `toString()`, you can just do:
```java
        return Objects.hash(getFqdn(), getRootCa(), getMirrorCredentials());
```